public/swagger-ui/css/print.css

```diff
@@ -0,0 +1,1367 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
Also, it is important to have an easy way of upgrading the dependency in the future as new versions are released -- the more automated we can make that procedure, the better.